site/content/en/_index.md

```diff
@@ -19,12 +19,15 @@ These functions can be implemented using any toolchain such as the [KPT Function
 
 See [definition of source functions][source].
 
-| Image                              | Args              | Description                                                                                             | Source                                                                                                           |
-| ---------------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
-| gcr.io/kpt-dev/kpt                 | fn source         | Reads a directory of Kubernetes configuration recursively.                                              | [Link](https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/source.go)          |
-| gcr.io/kpt-functions/read-yaml     |                   | [Demo] Reads a directory of Kubernetes configuration recursively.                                       | [Link](https://github.com/GoogleContainerTools/kpt-functions-sdk/tree/master/ts/demo-functions/src/read_yaml.ts) |
-| gcr.io/cloud-builders/kubectl      | get [...] -o yaml | Get one or many resources from a Kubernetes cluster.                                                    | [Link](https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/kubectl/Dockerfile)                     |
-| gcr.io/kpt-functions/helm-template |                   | Render chart templates locally using helm template. (See [usage](docs/helm-template/usage/index.html)). | [Link](https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/helm-template/run.sh) |
+| Image                                          | Args              | Description                                                                                             | Source                                                                                                                                         |
```

Comment:
I think we're missing a column here for the toolchain/SDK used to create these functions. It should link to appropriate SDK documentation if users want to develop their own (Note: Some are not using a SDK per, e.g. kubectl)

Reply:
That's a good suggestion. I'd rather clean up the sdk documentation first and then add links wholesale once the docs are in better shape. I added a note to our existing documentation issue (https://github.com/GoogleContainerTools/kpt/issues/640).

Reply:
sure.